include/bluetooth/mesh/vnd/dm_cli.h

```diff
@@ -0,0 +1,223 @@
+ *
+ */
+#define BT_MESH_MODEL_DM_CLI_INIT(_arr, _cnt, _handlers)                                           \
+	{                                                                                          \
+		.res_arr = _arr,                                                                   \
+		.entry_cnt = _cnt,                                                                 \
+		.handlers = _handlers,								   \
+	}
+
+/** @def BT_MESH_MODEL_DM_CLI
+ *
+ * @brief Distance measurement Client model entry.
+ *
+ * @param[in] _cli Pointer to a @ref bt_mesh_dm_cli instance.
+ */
+#define BT_MESH_MODEL_DM_CLI(_cli)                                                       \
+	BT_MESH_MODEL_VND_CB(BT_MESH_VENDOR_COMPANY_ID, BT_MESH_MODEL_ID_DM_CLI,         \
+			     _bt_mesh_dm_cli_op, &(_cli)->pub,                           \
+			     BT_MESH_MODEL_USER_DATA(struct bt_mesh_dm_cli, _cli),       \
+			     &_bt_mesh_dm_cli_cb)
+
+/** @brief Set or get the config state in the srv.
+ *
+ * This call is blocking if the @p rsp buffer is non-NULL. Otherwise, this
+ * function will return, and the response will be passed to the
+ * @ref bt_mesh_dm_cli_handlers::cfg_status_handler callback.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] set New configuration parameters, or NULL to get the current parameters.
+ * @param[out] rsp Status response buffer, or NULL to keep from blocking.
+ *
+ * @retval 0 Successfully sent the message and populated the @p rsp buffer.
+ * @retval -EALREADY A blocking request is already in progress.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ * @retval -ETIMEDOUT The request timed out without a response.
+ */
+int bt_mesh_dm_cli_config(struct bt_mesh_dm_cli *cli,
+				    struct bt_mesh_msg_ctx *ctx,
+				    const struct bt_mesh_dm_cfg *set,
+				    struct bt_mesh_dm_cli_cfg_status *rsp);
+
+/** @brief Start a distance measurement on a srv.
+ *
+ * This call is blocking if the @p rsp buffer is non-NULL. Otherwise, this
+ * function will return, and the response will be passed to the
+ * @ref bt_mesh_dm_cli_handlers::result_handler callback.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] set Distance measurement Start parameters.
+ * @param[out] rsp Status response buffer, or NULL to keep from blocking.
+ *
+ * @retval 0 Successfully sent the message and populated the @p rsp buffer.
+ * @retval -EALREADY A blocking request is already in progress.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ * @retval -ETIMEDOUT The request timed out without a response.
```

Comment:
```suggestion
 * @retval -ETIMEDOUT The request timed out without a response.
 * @retval -EBADMSG The supplied configuration parameters are invalid.
```

Same in `bt_mesh_dm_cli_config`